common/datasets/librispeech.py

```diff
@@ -349,8 +349,18 @@ def get_bliss_lexicon(subdir_prefix=""):
         checksum="d722bc29908cd338ae738edd70f61826a6fca29aaa704a9493f0006773f79d71",
     )
 
+    text_lexicon = download_lexicon_job.out_file
+    if use_stress_marker:
+        from i6_core.text import PipelineJob
+        eliminate_stress_job = PipelineJob(
+        text_lexicon, ["sed 's/[0-9]//g'"],
```

Comment:
Are you sure it is safe to do this? what if there is something like: `<phon score=0.0>...`? and maybe there are other cases

Reply:
Please check the input to the Job. This is not a bliss lexicon, but a text file. Namely, the one you take after downloading the corpus.

Reply:
ah sorry i thought it is a bliss lexicon.

Reply:
I see. I thought about this because in SWB lexicon text file for example, there are orths with digits.

Reply:
what if words contain digits? (e.g. "9/11" or "hdf5")

Reply:
ok i checked seems to be no words with digits.

Reply:
> ok i checked seems to be no words with digits.

Glad you are now convinced, even though I already told you that this has been checked by me.